bundles/org.openhab.binding.senseenergy/src/main/java/org/openhab/binding/senseenergy/internal/actions/SenseEnergyMonitorActions.java

```diff
@@ -105,7 +102,7 @@ Map<String, QuantityType<?>> queryEnergyTrend( //
         SenseEnergyApiGetTrends trends;
         try {
             trends = localDeviceHandler.getApi().getTrendData(localDeviceHandler.getId(), trendScale, localDateTime);
-        } catch (InterruptedException | TimeoutException | ExecutionException | SenseEnergyApiException e) {
+        } catch (Exception e) {
```

Comment:
Why changed this, we prefer the catch of explicit exceptions over the generic `Exception`